<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>245</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>268</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>66</y>
      <width>373</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_20">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_30">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_28">
       <property name="text">
        <string>x²</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_26">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_24">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_25">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="pushButton_29">
       <property name="text">
        <string>√</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_27">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_21">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="pushButton_18">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_19">
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="pushButton_23">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_22">
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_31">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_32">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_33">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_34">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QPushButton" name="pushButton_35">
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalculator">
    <property name="title">
     <string>Calculator</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPast"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuCalculator"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/files/images/copy.svg</normaloff>:/files/images/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionPast">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/files/images/paste.svg</normaloff>:/files/images/paste.svg</iconset>
   </property>
   <property name="text">
    <string>Past</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/files/images/undeo.svg</normaloff>:/files/images/undeo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/files/images/redo.svg</normaloff>:/files/images/redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/files/images/circle-xmark.svg</normaloff>:/files/images/circle-xmark.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
